src/helpers/Blocks/Blocks.test.js

```diff
@@ -639,5 +663,48 @@ describe('Blocks', () => {
         'has--borderRadius--8',
       ]);
     });
+
+    it('Understands noprefix converter for style values', () => {
+      const styles = {
+        color: 'red',
+        'theme:noprefix': 'primary',
+      };
+      expect(buildStyleClassNamesFromData(styles)).toEqual([
+        'has--color--red',
+        'primary',
+      ]);
+    });
+
+    it('Understands bool converter for trueish value', () => {
+      const styles = {
+        color: 'red',
+        'inverted:bool': true,
+      };
+      expect(buildStyleClassNamesFromData(styles)).toEqual([
+        'has--color--red',
+        'inverted',
+      ]);
+    });
+
+    it('Understands bool converter for false value', () => {
+      const styles = {
+        color: 'red',
+        'inverted:bool': false,
+      };
+      expect(buildStyleClassNamesFromData(styles)).toEqual([
+        'has--color--red',
+        '',
+      ]);
+    });
+
+    it('Ugly edge cases', () => {
```

Comment:
I don't know if fixing this edge case would be breaking. @sneridagh ?

Reply:
I'd say it's a bugfix, not in the spec of the feature.

Reply:
Fixed